pkg/internal/netolly/agent/pipeline_test.go

```diff
@@ -52,7 +52,7 @@ func TestFilter(t *testing.T) {
 			},
 			Attributes: beyla.Attributes{Select: attributes.Selection{
 				attributes.BeylaNetworkFlow.Section: attributes.InclusionLists{
-					Include: []string{"beyla_ip", "direction", "dst_port", "iface", "src_port", "transport"},
+					Include: []string{"beyla_ip", "iface.direction", "dst_port", "iface", "src_port", "transport"},
```

Comment:
just confirming this is not meant to be `iface_direction`.

Reply:
Ah! Good point. I meant `iface_direction` but, to facilitate our users' life, Beyla accepts as a configuration both the OTEL format (`iface.direction`) and Prometheus format (`iface_direction`). That's why the tests still passed.